### PR TITLE
Upgrading async to 3.2.2

### PR DESCRIPTION
**Pull request by sharmakhushboo**
Upgrading async to fix this vulnerability https://www.whitesourcesoftware.com/vulnerability-database/CVE-2021-43138

### CODE REVIEW

**Review by tylerbutler — COMMENTED** [review]
There's also a reference in the lockfile for the getkeys tool: https://github.com/microsoft/FluidFramework/security/dependabot/220

**Review by hedasilv — COMMENTED** [review]
FYI @sharmakhushboo, looks like dependabot is also sending PRs to upgrade `async` to 3.2.2:

Some examples:
https://github.com/microsoft/FluidFramework/pull/9877
https://github.com/microsoft/FluidFramework/pull/9876

**Comment by znewton** [comment]
It might be worth upgrading @types/async to ^3.2.9 which was released around when async 3.2.2 was released

**Comment by sharmakhushboo** [comment]
> There's also a reference in the lockfile for the getkeys tool: https://github.com/microsoft/FluidFramework/security/dependabot/220

@tylerbutler async in getKeys is coming from ms-rest-azure package. This package has not been updated with the latest version of async yet. I'll still upgrade 'ms-rest-azure' to latest in getKeys though.

side note- i was not able to access the link you shared.

File: server/historian/packages/historian/package.json [hunk 1]
```diff
@@ -56,7 +56,7 @@
     "@types/uuid": "^3.4.4",
     "@typescript-eslint/eslint-plugin": "~5.9.0",
     "@typescript-eslint/parser": "~5.9.0",
-    "async": "^3.2.0",
```
Comment (znewton) [inline server/historian/packages/historian/package.json:59]: These changes should be reflected in server/historian/lerna-package-lock.json

File: server/gitrest/packages/gitrest/package.json [hunk 2]
```diff
@@ -52,7 +52,7 @@
     "@fluidframework/build-common": "^0.23.0",
     "@fluidframework/eslint-config-fluid": "^0.28.1000-61189",
     "@rushstack/eslint-config": "^2.5.1",
-    "@types/async": "^3.2.6",
```
Comment (znewton) [inline server/gitrest/packages/gitrest/package.json:55]: These changes should be reflected in server/gitrest/lerna-package-lock.json

**Pull request closed without merging**